src/linux/mod.rs

```diff
@@ -26,11 +26,11 @@ pub struct LinuxSandbox {
 impl Sandbox for LinuxSandbox {
     fn new() -> Result<Self> {
         // Setup landlock filtering.
-        let mut landlock = Ruleset::new()
+        let landlock = Ruleset::new()
             .set_best_effort(false)
             .handle_access(AccessFs::from_all(ABI))?
-            .create()?;
-        landlock.set_no_new_privs(true);
+            .create()?
+            .set_no_new_privs(true);
```

Comment:
I'm not sure I like this tbh. I don't think having `set_no_new_privs` as builder pattern after `create` is a good look.

Reply:
I've reverted this to use the new `.as_mut()`.